ports/libxml2/portfile.cmake

```
@@ -21,6 +34,10 @@ vcpkg_check_features(OUT_FEATURE_OPTIONS FEATURE_OPTIONS
         "tools" LIBXML2_WITH_PROGRAMS
         "icu"  LIBXML2_WITH_ICU
 )
+
+vcpkg_find_acquire_program(PKGCONFIG)
+set(ENV{PKG_CONFIG} "${PKGCONFIG}")
+
```

```suggestion

vcpkg_find_acquire_program(PKGCONFIG)
```
It is recommended to add option "-DPKG_CONFIG_EXECUTABLE=${PKGCONFIG}" in vcpkg_cmake_configure to pass in pkgconfig.

---

Fixed!